test/operators/second_quantization/test_fermionic_op.py

```diff
@@ -141,6 +141,16 @@ def test_init_from_tuple_label(self):
         ]
         self.assertListEqual(actual.to_list(), desired)
 
+    def test_iter(self):
```

Comment:
Could you check the type of iter(op)? If it returns `_data`, this past will be passed.